lib/Controller/DraftsController.php

```diff
@@ -208,7 +208,7 @@ public function update(int $id,
 	 */
 	#[TrapError]
 	public function destroy(int $id): JsonResponse {
-		$message = $this->service->getMessage($id, $this->userId);
+		$message = $this->service->getMessage($id, $this->userId, LocalMessage::TYPE_DRAFT);
```

Comment:
Nit: I'd prefer to use `\OCA\Mail\Service\OutboxService::getMessage()` to fetch outbox messages and `\OCA\Mail\Service\DraftsService::getMessage()` to fetch drafts.

This also applies to the other usages.